php/class-customize-snapshot-manager.php

```diff
@@ -1469,6 +1470,26 @@ public function render_templates() {
 				<p>{{ data.message }}</p>
 			</div>
 		</script>
+
+		<script type="text/html" id="tmpl-snapshot-conflict">
+			<?php $title_text = sprintf( esc_html__( 'Potential Snapshot conflicts (click to expand)', 'customize-snapshots' ) );?>
+			<# id= data.setting_id.replace( /]/g, '' ).split( '[' ).filter( Boolean ).join( '-' ); #>
+			<a href="#TB_inline?width=600&height=550&inlineId=snapshot-conflicts-{{id}}" class="dashicons dashicons-warning thickbox snapshot-conflicts-thickbox" title="<?php echo $title_text; ?>"></a>
+			<div id="snapshot-conflicts-{{id}}" style="display:none;">
```

Comment:
`thickbox` is initially hidden on click of conflict icon it should open Thickbox. I will move this css to css file.